Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG GO_VERSION=1.18
+ARG ALPINE_VERSION=3.16
+
+### Build binary
+FROM golang:${GO_VERSION}-alpine${ALPINE_VERSION} as build-binary
+ENV GOOS=linux \
+    GOARCH=amd64 \
+    CGO_ENABLED=0
+COPY . /go/src/github.com/deepmap/oapi-codegen
+WORKDIR /go/src/github.com/deepmap/oapi-codegen
+RUN go build -o ./bin/oapi-codegen ./cmd/oapi-codegen/oapi-codegen.go
+
+### Image
+FROM alpine:${ALPINE_VERSION} as image
+COPY --from=build-binary /go/src/github.com/deepmap/oapi-codegen/bin/oapi-codegen /usr/local/bin/oapi-codegen
+RUN chmod +x /usr/local/bin/oapi-codegen
+ENTRYPOINT ["/usr/local/bin/oapi-codegen"]
```

Comment:
Is it possible for this to be `FROM scratch`?

Reply:
Done!